src/components/inputs/text/index.tsx

```diff
@@ -0,0 +1,14 @@
+    
+import { InputHTMLAttributes } from "react";
+
+import styles from './TextInput.module.scss'
+
+interface Props extends InputHTMLAttributes<HTMLInputElement> {
+    t?: unknown
```

Comment:
use `type Props = InputHTMLAttributes<HTMLInputElement>` if you have no properties for extension